Utils/src/METDouble.cc

```diff
@@ -142,7 +142,7 @@ METDouble::produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSe
    double dpnhat[3];
    unsigned int goodcount=0;
    
-   for(int i=0; i<3; ++i)dpnhat[i]=-999;
+   for(double & i : dpnhat)i=-999;
```

Comment:
Why not 'auto' here?

Reply:
auto is often not used for primitive types

Reply:
Really. I didn't know that. That's good information to have.

Reply:
The goal of auto is to improve readability. Obviously this is somewhat in the eye of the beholder, but while replacing some 50-character iterator type with auto definitely improves readability, replacing int/double/float etc. probably doesn't.